src/core/Form/Select/SingleSelect/SingleSelect.tsx

```diff
@@ -335,13 +427,13 @@ class BaseSingleSelect<T> extends Component<
       focusedDescendantId,
       filterInputValue,
       selectedItem,
+      computedItems,
     } = this.state;
 
     const {
       id,
       className,
       theme,
-      items: propItems,
```

Comment:
Removing this leads to the item list being leaked into the DOM.

Reply:
Good catch. Added it back in, although `items` is no longer used in rendering because there is now `computedItems` state.